src/modules/workflows/workflows.js

```diff
@@ -239,43 +232,95 @@ router.get('/:id', async (req, res) => {
   }
 });
 
-// Update workflow status and assign writer or editor
-router.patch('/:id', validateMiddleware({ body: updatedWorkflowDto }), async (req, res) => {
+// Update workflows status and assign writer or editor
+router.patch('/assign', validateMiddleware({ body: assignWorkflowDto }), async (req, res) => {
   try {
-    const { id } = req.params;
-    const { ...updatedFields } = req.body;
+    const {
+      filters,
+      assignments: { writer, editor }
+    } = req.body;
```

Comment:
How are you getting workflow id from UI ?

Reply:
from filters and it can be anything based on filters will fetch workflows and bulk assign